src/rest.jl

```diff
@@ -184,6 +184,6 @@ function request(
     isnothing(body) && (body = UInt8[])
     headers = _ensure_headers(headers)
     _authenticate!(ctx, headers)
-    opts = (;redirect = false, connection_limit = 4096)
+    opts = (;redirect = false, pool = HTTP.Pool(4096))
```

Comment:
Ah shoot, I should fix the warning message in HTTP. By passing it like this, we would unfortunately be creating a _new_ connection pool to be used for _each_ request. Instead, we want to create a single global `const POOL = HTTP.Pool(4096)`, then pass `pool=POOL` so the _same_ pool is used across all the requests. Sorry about that.

(but thanks for doing this! I just noticed we were hitting this warning too so I was going to come update as well)

Reply:
that makes sense - the invocation did have a slightly off code smell to me, but I didn't dive into it at all. Will you take over updating the sdk to the best option?

Reply:
Can do